CMSgov/mint-app
# [EASI-4080] Convert Collaborators folder and NDA Queries

# EASI-4080


## Changes and Description

- Convert GetIsCollaborator
- Convert GetModelCollaborators
- convert GetModelPlanCollaborator
- Convert CreateModelPlanCollaborator
- Convert CreateModelPlanCollaborator and DeleteModelPlanCollaborator
- Convert UpdateNDA



## How to test this change

1. `scripts/dev gql`
2. Check to see if updating NDA and adding/rendering collaborator works as expected



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
